include/z64actor.h

```diff
@@ -357,4 +357,101 @@ typedef enum {
     DOORLOCK_NORMAL_SPIRIT
 } DoorLockType;
 
+typedef enum {
+    NAVI_ENEMY_DEFAULT,
+    NAVI_ENEMY_GOHMA,
+    NAVI_ENEMY_ENGOMA_EGG,
+    NAVI_ENEMY_ENGOMA,
```

Comment:
ENGOMA is GOHMA_LARVA. and i would change the EGG one to be the English name instead of internal name

Reply:
+ `NAVI_ENEMY_GOHMA` -> `NAVI_ENEMY_QUEEN_GOHMA` or "boss" may be more explicit

Reply:
The only time she is called Queen Gohma is when the deku tree calls her that. The navi enemy textbox itself only says gohma, so im inclined to go with that

![image](https://user-images.githubusercontent.com/22045047/162961553-492875f1-8b65-4061-a028-e6d667adc6fa.png)
![image](https://user-images.githubusercontent.com/22045047/162961588-853ee1cb-a28f-44ad-8b98-27a0f55813c4.png)